README.md

```diff
@@ -1,24 +1,13 @@
-GitHub Enterprise Backup Utilities
-==================================
+# GitHub Enterprise Backup Utilities
 
 This repository includes backup and recovery utilities for [GitHub Enterprise][1].
 
-- **[Features](#features)**
-- **[Requirements](#requirements)**
-  - **[Backup host requirements](#backup-host-requirements)**
-  - **[Storage requirements](#storage-requirements)**
-  - **[GitHub Enterprise version requirements](#github-enterprise-version-requirements)**
-- **[Getting started](#getting-started)**
-- **[Migrating from GitHub Enterprise v11.10.34x to v2.0](#migrating-from-github-enterprise-v111034x-to-v20-or-v21)**
-- **[Using the backup and restore commands](#using-the-backup-and-restore-commands)**
-- **[Scheduling backups](#scheduling-backups)**
-- **[Backup snapshot file structure](#backup-snapshot-file-structure)**
-- **[How does backup utilities differ from a High Availability replica?](#how-does-backup-utilities-differ-from-a-high-availability-replica)**
-- **[Support](#support)**
+**Note**: the [GitHub Enterprise version requirements](docs/requirements.md#github-enterprise-version-requirements) have
+changed starting with Backup Utilities v2.13.0.
```

Comment:
Should we add a date here so folks have a better sense of when this requirement changed?

Reply:
Good idea (for those who follow master so don't get versions).